spyder/plugins/ipythonconsole/widgets/debugging.py

```diff
@@ -584,9 +575,6 @@ def _pdb_readline_callback(self, line):
 
     def pdb_input(self, prompt, password=None):
         """Get input for a command."""
-        if self._hidden:
-            raise RuntimeError(
-                'Request for pdb input during hidden execution.')
```

Comment:
It this related to the changes in jupyter/qtconsole#505?

Reply:
Yes because self._hidden is only meaningful with relation to a specific message which we don't have access to here.

Reply:
Ok, I'll merge that PR and release 5.1.2 with the fix so that our tests pass.